bundle/deploy/terraform/convert_test.go

```diff
@@ -455,6 +455,20 @@ func TestBundleToTerraformModelServingPermissions(t *testing.T) {
 	var src = resources.ModelServingEndpoint{
 		CreateServingEndpoint: &serving.CreateServingEndpoint{
 			Name: "name",
+
+			// Need to specify this to satisfy the equivalence test:
+			// The previous method of generation includes the "create" field
+			// because it is required (not marked as `omitempty`).
+			Config: serving.EndpointCoreConfigInput{
+				ServedModels: []serving.ServedModelInput{
+					{
+						ModelName:          "model_name",
+						ModelVersion:       "1",
+						ScaleToZeroEnabled: true,
+						WorkloadSize:       "Small",
+					},
+				},
+			},
```

Comment:
Comment explains why this is needed.

Reply:
Could you extend the comment to mention that the previous generation method was based on the std library json marshaller, and the current method based on dyn.Value omits empty structs when refs are missing.

Otherwise, this might not make sense to a reader without the context of this PR.

Reply:
Done, thanks.